projects/demo/src/modules/components/breadcrumbs/index.ts

```diff
@@ -1,27 +1,18 @@
+import {NgFor} from '@angular/common';
 import {Component} from '@angular/core';
+import {RouterLink} from '@angular/router';
 import {changeDetection} from '@demo/emulate/change-detection';
-import type {TuiDocExample} from '@taiga-ui/addon-doc';
-import type {TuiSizeL} from '@taiga-ui/core';
+import {TuiDemoModule} from '@demo/utils';
+import {TuiLinkDirective, type TuiSizeL} from '@taiga-ui/core';
+import {TuiBreadcrumbsModule} from '@taiga-ui/kit';
 
 @Component({
-    selector: 'example-breadcrumbs',
-    templateUrl: './breadcrumbs.template.html',
+    standalone: true,
+    imports: [NgFor, RouterLink, TuiDemoModule, TuiBreadcrumbsModule, TuiLinkDirective],
```

Comment:
`TuiDemoModule` contains `CommonModule`, so `NgFor` is redundant

Reply:
update